smart-contracts/contracts/BridgeBank/IbcToken.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
@smartyalgo Correct. Once this PR gets merged in, I'll open a new PR to deal with IbcToken/BridgeToken.

Just for some context: we created this temporary IbcToken so that Michael and Brandon could run tests quickly. In `future/peggy2` the IbcToken doesn't exist anymore (it's called BridgeToken).